generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -183,7 +173,8 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting ofLength
        | null |
        | "♀♀" |
 
-Scenario: Running a 'shorterThan' request alongside a contradicting ofLength (too short) constraint should fail with an error message
+@ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501